timesource/timesource.go

```diff
@@ -28,6 +28,10 @@ const (
 
 	// DefaultRPCTimeout defines write deadline for single ntp server request.
 	DefaultRPCTimeout = 2 * time.Second
+
+	// TimeChangeThreshold defines the minimum time difference that indicates
+	// system time has been changed. Values smaller than this are considered normal drift.
+	TimeChangeThreshold = 1 * time.Second
```

Comment:
Could you elaborate and comment on source why 1 second seems like a good choice? It's normal that computers drift a few seconds per week if not synced with time servers. https://security.stackexchange.com/questions/227002/is-it-safe-to-assume-that-my-computers-clock-will-always-be-synced-with-actual

Reply:
Thank you and thanks bot ;)